Tests/DownloadTests.swift

```diff
@@ -252,7 +252,7 @@ class DownloadResponseTestCase: BaseTestCase {
         // Given
         let fileURL = randomCachesFileURL
         let urlString = "https://httpbin.org/get"
-        let parameters = ["foo": "bar"]
+        let parameters = ["foo": "bar" as AnyObject]
```

Comment:
Unfortunately that doesn't work: literal strings are no longer automatically cast to `AnyObject`. If we did that, it would have to be `let parameters: [String: AnyObject] = ["foo": "bar" as AnyObject]`.